File: JAVASCRIPT/SimpleArithmeticAppSelfTrial/SimpleArithmeticApp.js
    const prompt = require("prompt-sync")();

   let userScore = 0;
    

    let counter = 0;

    while (counter < 10){
        let userInput = 10;
        while (true){
            let firstNumber = Math.floor(Math.random() * 10);
            let secondNumber = Math.floor(Math.random() * 10);
            let difference = (firstNumber > secondNumber) ? (firstNumber - secondNumber) : (secondNumber - firstNumber);

                (firstNumber > secondNumber) ? console.log(firstNumber + "-" + secondNumber + " " + "=" + " " + "?") : console.log(secondNumber + "-" + firstNumber + " " + "=" + " " + "?");
                //console.log(firstNumber + "-" + secondNumber + " " + "=" + " " + "?");

            
            let userAttempt = 0;

            let userInput = Number(prompt("Guess The Answer: "));

            if (userInput == difference){
                console.log("Correct Guess: " + difference);
                userScore++;
            }
            else{
                console.log("Incorrect Answer!!");
                prompt("Guess The Answer: ");
                if (userInput == difference){
                    userAttempt++;
                    

               }
           }
       
        
           counter++; 
           if (counter == 10){
            break;
        }
        
        }
        
    }
 

     console.log("The total score of the user is: " + userScore); 
     //console.log(mytime);  

File: JAVA/SimpleArithmeticAppSelfTrial/SimpleArithmeticApp.java


import java.util.Scanner;
import java.util.Random;
import java.time.LocalTime;

public class SimpleArithmeticApp{

    public static void main (String [] args){
     

    Random random = new Random();
    LocalTime mytime = LocalTime.now();

    int userScore = 0;
    

    int counter = 0;

    while (counter < 10){
        int userInput = 10;
        while (true){
            int firstNumber = (int) (Math.random() * 10);
            int secondNumber = (int)(Math.random() * 10);
            int difference = (firstNumber > secondNumber) ? (firstNumber - secondNumber) : (secondNumber - firstNumber);


                System.out.println(firstNumber + "-" + secondNumber + " " + "=" + " " + "?");

            
            int userAttempt = 0;
            Scanner input = new Scanner (System.in);
            System.out.println("Guess The Answer: ");
            userInput = input.nextInt();

            if (userInput == difference){
                System.out.println("Correct Guess: " + difference);
                userScore++;
            }
            else{
                System.out.println("Incorrect Answer!!");
                System.out.println("Guess The Answer: ");
                if (userInput == difference){
                    userAttempt++;
                    

               }
           }
       
        
           counter++; 
           if (counter == 10){
            break;
        }
        
        }
        
    }
    
    

        
          
        
        
        
        

     System.out.println("The total score of the user is: " + userScore); 
     System.out.println(mytime);  
    }
    
   
    
    
      




}




